EDI/Operation/02_ManageHeaderFileSett.md

```diff
@@ -0,0 +1,404 @@
+    - The *S_BGM C_C106* segment is now highlighted with a small black border. This indicates that the segment contains data.   
+    - The string attribute has been marked for a later data mapping.
+    
+5. Click the [SAVE] button to save the message structure.   
+    The *Exporter definitions* view is displayed. All defined string attributes are added to the attribute mapping function in the *Data sources* tab.
+
+6. If desired, check if the string attribute has been really marked for a later data mapping. To do this, open the message definition again and click the *Data sources* tab.  
+
+7. Select the related data source.   
+    - The *Map attributes* wizard window is displayed.
+    - The EDI Export BGM/C106/1004 *Destination attribute* has been added to the attributes to be mapped later or directly afterwards, if desired. For detailed information, see [Map data source](./05_ManageDataSources.md#map-data-source). 
+
+       ![Map attributes](../../Assets/Screenshots/EDI/Operation/ExportDefBGM_C106Mapping.png "[Map Attributes]")
+
+8. Finalize the *Map attributes* wizard by clicking the [CONTINUE] buttons until you can click the [FINALIZE] button  
+   The *Exporter definition sources* view is displayed.
+
+9. Continue with [Specify message dates](#specify-message-dates) described below.
+
+
+
+## Specify message dates
+
+Specify the message dates your business partner requires.   
+
+Each invoice can have a lot of different dates, for example the invoice date, the expected delivery date, the tax point date, and much more. The Actindo EDIFACT message structure contains only one BTM segment in its raw state. However, for many business partners, multiple BTM segments must be completed. As soon as you have specified one value in a BTM segment, a further BTM segment is added automatically so that you can specify as much BTM segments as your business partner require.
+
+This procedure shows how to edit message dates. The message dates require both the description of the date and a variable used to determine it.
+
+#### Prerequisites
+
+- You have expanded the message structure, see [Get header file segments](#get-header-file-segments).
+
+#### Procedure
+
+*DataHub Exporter > EXPORTS tab > Select message definition > File settings tab > Expand message structure*
+
+![Message structure](../../Assets/Screenshots/EDI/Operation/ExportDefMessDefDates.png "[Message structure]")
+
+1. Expand the *S_DTM* (DATE &frasl; TIME &frasl; PERIOD) segment by clicking the ![Collapsed](../../Assets/Icons/NodeCollapsed.png "[Collapsed]") (Collapsed) button at it.   
+    The *S_DTM* segment in turn consists of the subordinate *C_C507* segment.
+
+2. Expand the *C_C507* (Date and &frasl; or time, or period relevant to the) segment by clicking the ![Collapsed](../../Assets/Icons/NodeCollapsed.png "[Collapsed]") (Collapsed) button at it. In most cases it is required to specify this segment.   
```

Comment:
Oben, beim Beginning of message, hast du Bezug auf dem übergeordnete Segment genommen, also S_BGM C_C106. So was es deutlicher, finde ich. Kann man es auch hier auch machen?

Reply:
Wenn ich das mache, muss ich immer den gesamten Pfad aufschreiben. das verwirrt noch mehr